control/server.py

```diff
@@ -401,6 +401,18 @@ def _start_spdk(self, omap_state):
         for trtype in spdk_transports.split():
             self._create_transport(trtype.lower())
 
+        with self.ping_lock:
+            try:
+                ret_ver = spdk.rpc.spdk_get_version(self.spdk_rpc_ping_client)
+                try:
+                    ver_str = ret_ver["version"]
+                    self.logger.info(f"Started SPDK with version \"{ver_str}\"")
+                except KeyError:
+                    self.logger.error(f"Can't find SPDK version string in {ver_ret}")
```

Comment:
is there a typo in ver_ret (should be changed to  ret_ver I think)
Suggesting  to use full words, on that case return_version or any other meaning, also for ver_str

Reply:
Yes, nice catch.